### PR TITLE
[#18963] feat: actions UI for stranger contact profile

### LINKED ISSUE #18963: Feature: Actions for stranger contact profile (3 dot menu)
blocked by 
- #18961 

User story:

1. **Initiating Actions from the Profile Menu**
   - As a user, I want to click on the three-dot button to see additional profile actions, so that I can manage my interactions with this person.

1. **Adding a Nickname**
   - As a user, I want to add a nickname to the contacts in my network, so that I can remember or identify them more easily in my conversations.

1. **Viewing a QR Code**
   - As a user, I want to display my own QR code or that of others, so that I can share profiles quickly via a visual code scan.

1. **Sharing a User Profile**
   - As a user, I want the option to share someone's profile, so that I can refer them to my connections or across different platforms.

1. **Marking as Untrusted**
   - As a user, I want to mark a profile as untrusted, so that I can be cautious in future interactions and maintain my online safety.

1. **Blocking a User**
   - As a user, I want to block someone, so that I no longer receive messages from them and they are not able to see mine in mutual groups.

Scope: 
- 3 dot button
- the menu
- options in menu lead to nothing

Figma https://www.figma.com/file/jGs1P4RU4R1NRrY0xXmHC1/Profile-interactions?type=design&node-id=1-66300&mode=design&t=L27O8eaCfS59gCGl-4

Image

### PR DESCRIPTION
**Pull request by mohsen-ghafouri**
fixes #18963

### Summary

Actions UI for stranger contact profile

### Result

status: ready

### CODE REVIEW

File: src/status_im/contexts/profile/contact/view.cljs [hunk 1]
```diff
@@ -20,5 +20,8 @@
       :cover-color      (or customization-color colors/primary-50)
       :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
       :page-nav-props   {:right-side [{:icon-name :i/options
-                                       :on-press  not-implemented/alert}]}}
+                                       :on-press  (fn []
+                                                    (rf/dispatch [:show-bottom-sheet
+                                                                  {:content (fn []
+                                                                              [actions/view])}]))}]}}
```
Comment (yqrashawn) [inline src/status_im/contexts/profile/contact/view.cljs]: Not sure if this should be wrapped in use-effect like the ui guideline says.

**Review by yqrashawn — APPROVED** [review]
Left one comment. LGTM other than the comment.

File: src/status_im/contexts/profile/contact/view.cljs (hunk 1 above)
Comment (seanstrom) [reply src/status_im/contexts/profile/contact/view.cljs]: Yeah I think callbacks need to be wrapped in a `use-callback` function, and that needs to be sorta structured like this example:
```
(defn view
  []
  (let [on-press (rn/use-callback 
                  (fn [] 
                    (rf/dispatch [:show-bottom-sheet 
                                  {:content (fn [] [actions/view])}])))]
    [rn/pressable
     {:on-press on-press}]))
```

**Review by seanstrom — COMMENTED** [review]
Looks good! ✅ 
I left a similar comment about needing a `use-callback`

File: src/status_im/contexts/profile/contact/view.cljs (hunk 1 above)
Comment (mohsen-ghafouri) [reply src/status_im/contexts/profile/contact/view.cljs]: resolved please check again @seanstrom

**Review by seanstrom — APPROVED** [review]
Wooh! Looks good ✅

**Comment by mohsen-ghafouri** [comment]
Hey @status-im/mobile-qa just a confirmation if this PR requires any manual QA or not. the only changes related to the current screens is a translation changes `Block User` - > `Block user`.

**Comment by pavloburykh** [comment]
> Hey @status-im/mobile-qa just a confirmation if this PR requires any manual QA or not. the only changes related to the current screens is a translation changes `Block User` - > `Block user`.

Hey @mohsen-ghafouri! Thanks for the PR. E2E failures are not PR related, so we can merge it.

**Pull request merged**